packages/sommelier.rb

```diff
@@ -393,22 +364,36 @@ def self.install
 
   def self.postinstall
     # all tasks are done by sommelier.env now
-    puts
-    puts 'Removing old sommelier env variable loading code in ~/.bashrc'.lightblue
-    system 'sed -i "/[sS]ommelier/d" -i.backup ~/.bashrc'
-    puts 'To complete the installation, execute the following:'.orange
-    puts 'source ~/.bashrc'.orange
+    now = `date +%Y%m%d%H%M`.chomp
+    FileUtils.cp "#{HOME}/.bashrc", "#{HOME}/.bashrc.#{now}"
+    system "sed -i '/[sS]ommelier/d' #{HOME}/.bashrc"
+    diff = `diff #{HOME}/.bashrc #{HOME}/.bashrc.#{now}`.chomp
```

Comment:
The previous command will fail if the file is not modified properly, extra checking is not needed

Reply:
The checking is there because the message is not needed if there are no changes.